src/index.ts

```diff
@@ -33,6 +55,7 @@ export function getSidebar(options: Options = {}) {
   options.contentDirs = options?.contentDirs?.length ? options.contentDirs : ['/'];
   options.collapsible = options?.collapsible ?? true;
   options.collapsed = options?.collapsed ?? true;
+  options.useFrontmatter = options?.useFrontmatter ?? false;
```

Comment:
@nandenjin can you check this suggestion?

Reply:
@JonathanSchndr I can do so, but I prefer to define new type to set, or use type decleration from `vitepress` (we should add it as peerDeps), because in the sugeested code, the return type defined by internal function.

In the first place it seems not about the changes in this PR - so I think it is a kind of bug of coderabbitai - but I can do some of these if you requests 👍